README.md

```diff
@@ -1,2 +1,22 @@
 # watson-waste-sorter
 ***Work in progress***
+
+# To Use
+
+First provision a Free tier [Visual Recognition](https://console.bluemix.net/catalog/services/visual-recognition) 
+Service and name it `visual-recognition-wws`.
+
+## On Cloud Foundry
+```
+cf push
+```
+
+## Running on Local
+```
+python run.py
+```
+
+## Backend API usage
+
+Do a Post request at `https://watson-waste-sorter.mybluemix.net/api/sort` with the image as the parameter. 
```

Comment:
s/Post/POST

Reply:
Can you post an actual sample request with the image as a parameter? It's not clear what the name of the parameter is